### PR TITLE
feat(UIForms): toggle widget

### PR DESCRIPTION
**Pull request by jsomsanith**
**What is the problem this PR is trying to solve?**
*WARNING : this is only for the next version of forms*
No Toggle widget in UIForms

**What is the chosen solution to this problem?**
Migrate Toggle widget

**Please check if the PR fulfills these requirements**
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/forms/src/UIForm/fields/Toggle/Toggle.component.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import React, { PropTypes } from 'react';
+import Toggle from 'react-talend-components/lib/Toggle';
+import FieldTemplate from '../FieldTemplate';
+
+const ToggleWidget = (props) => {
```
Comment (talend-glorieux) [inline packages/forms/src/UIForm/fields/Toggle/Toggle.component.js]: Please use a regular function instead of an arrow one.

Comment (jsomsanith) [inline packages/forms/src/UIForm/fields/Toggle/Toggle.component.js]: Oh sorry :( copy/paste daemon has been here

File: packages/components/src/Toggle/__snapshots__/Toggle.snapshot.test.js.snap [hunk 2]
```diff
@@ -8,6 +8,7 @@ exports[`Toggle should render a Checkbox 1`] = `
     htmlFor="id"
   >
     <input
+      autoFocus={undefined}
```
Comment (jmfrancois) [inline packages/components/src/Toggle/__snapshots__/Toggle.snapshot.test.js.snap]: use default value to get false here

Comment (jsomsanith) [inline packages/components/src/Toggle/__snapshots__/Toggle.snapshot.test.js.snap]: What is the benefit ? In DOM using undefine will just not add the attribute. With `false`, we will always have the attribute, most of the time with `false` value.

**Pull request merged**